### PR TITLE
feat: added notes and alias options for creating client ids

### PR DESCRIPTION
**Pull request by Michael-Vol**
**What kind of change does this PR introduce?** 
This PR adds support for adding optional alias and notes fields when creating client ids in security. It addresses the issue when there are multiple client ids and cannot be distinguished one from another solely from the id.
Solves #192.

- [ ] Bugfix
- [x] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other (please describe)

**Does this PR introduce a breaking change?** 

- [ ] Yes
- [x] No

**The PR fulfills these requirements:**

- [x] It's submitted to the `main` branch
- [x] When resolving a specific issue, it's referenced in the PR's description  (e.g. `fix #xxx`, where "xxx" is the issue number)

If adding a **new feature**, the PR's description includes:

- [x] A convincing reason for adding this feature

### CODE REVIEW

File: packages/security/src/admin/routes/UpdateSecurityClient.route.ts [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+      alias: ConduitString.Optional,
+      notes: ConduitString.Optional,
+    }),
+    async (params: ConduitRouteParameters) => {
+      const { platform, domain, alias, notes } = params.params!;
+      let client = await Client.getInstance().findOne({
+        _id: params.params!.id,
+      });
+      if (isNil(client)) {
+        throw new ConduitError('INVALID_PARAMS', 400, 'Client id not found');
```
Comment (kon14) [inline packages/security/src/admin/routes/UpdateSecurityClient.route.ts:46]: Rephrase this as "Security client not found" or something.

**Review by kon14 — CHANGES_REQUESTED** [review]

**Review by kon14 — APPROVED** [review]

File: packages/security/src/admin/routes/UpdateSecurityClient.route.ts [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+export function getUpdateSecurityClientRoute() {
+  return new ConduitRoute(
+    {
+      path: '/security/client/:id',
+      urlParams: {
+        id: ConduitString.Required,
+      },
+      action: ConduitRouteActions.UPDATE,
+      bodyParams: {
+        platform: ConduitString.Optional,
```
Comment (kkopanidis) [inline packages/security/src/admin/routes/UpdateSecurityClient.route.ts:25]: platform is not an acceptable argument. As we've discussed the user cannot change the platform

File: packages/security/src/admin/routes/UpdateSecurityClient.route.ts [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+        domain: ConduitString.Optional,
+        alias: ConduitString.Optional,
+        notes: ConduitString.Optional,
+      },
+    },
+    new ConduitRouteReturnDefinition('UpdateSecurityClient', {
+      id: ConduitString.Required,
+      clientId: ConduitString.Required,
+      clientSecret: ConduitString.Required,
```
Comment (kkopanidis) [inline packages/security/src/admin/routes/UpdateSecurityClient.route.ts:34]: client secret should not be returned after client creation

**Review by kkopanidis — CHANGES_REQUESTED** [review]

**Review by kkopanidis — APPROVED** [review]
